apps/src/gamelab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -108,7 +108,9 @@ export default connect(state => ({
     dispatch(pickLibraryAnimation(animation));
   },
   onUploadStart(data) {
-    if (data.files[0].type === 'image/png' || data.files[0].type === 'image/jpeg') {
+    if (data.files[0].size >= 101000) {
```

Comment:
Might be worth a comment about why this number (or alternatively at the top of the file do something like `const MAX_UPLOAD_SIZE = 101000;` and include a comment there about why this is 101kb)